crawler/main.js

```diff
@@ -75,7 +75,17 @@ configPromises.push(dataStore.configure({
 }));
 
 configPromises.push(urlJobProcessor.configure({
-
+  probes: [
+    {
+      name: 'totalRecords',
```

Comment:
This is a way to have the output reflect the number of sites that were successfully crawled. It was also the first working probe so it let me test the probe mechanism. There are probably better ways to get this info.